### ISSUE TITLE
Difference between indexName and logIndexName

### ISSUE THREAD

**Issue by lewismc** [mudrod/mudrod#87]
Hi @quintinali can you please explain the difference between the ```indexName``` and ```logIndexName``` properties in [config.xml](https://github.com/mudrod/mudrod/blob/master/core/src/main/resources/config.xml#L25-L26)?
```
  <para name="indexName">mudrod</para>
  <para name="logIndexName">podaaclog</para>
```
The reason I ask is because, as you probably know, two indexes are created when running the Mudrod pipeline... however the ```mudrod``` index is not utilized. Thanks

**Comment by lewismc** [comment]
@quintinali ping? Any ideas about this?

**Comment by quintinali** [comment]
@lewismc before we stored all the logs in different types in one index named "mudrod", however, with the increase of logs, the size of "mudrod" index grows as well which lead to search/index performance loss. So I modified codes to store monthly logs into different indexes. "logIndexName" is the prefix of log index. 

In a nutshell:

Information such as metadata, ontology are stored in index named "mudrod"(indexName)
PO.DAAC logs are stored in indexes, e.g. podaaclog201401, podaaclog201402 (logIndexName)
